### PR TITLE
feat: introduce slack command to get google auth url

### PR DESCRIPTION
**Pull request by dzehnder**
To remove the dependency of asking me and Alina for new google auth links, this will help to obtain the link for everyone

### CODE REVIEW

**Review by ekremney — APPROVED** [review]

File: test/support/slack/commands/create-google-link.test.js [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+        getIsLiveToggledAt: () => '2014-10-17T13:44:00.000Z',
+      }),
+    };
+
+    context = { dataAccess: dataAccessStub, log: console };
+    slackContext = { say: sinon.spy() };
+  });
+
+  describe('Initialization and BaseCommand integration', () => {
+    it('handles valid input and retrieves site status', async () => {
```
Comment (iuliag) [inline test/support/slack/commands/create-google-link.test.js:41]: test title doesn't correspond to the implementation? should it rather be "compiles the google link command correctly"?

File: test/support/slack/commands/create-google-link.test.js [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+      expect(slackContext.say.calledWith('https://spacecat.experiencecloud.live/api/ci/auth/google/123')).to.be.true;
+    });
+
+    it('responds with usage instructions for invalid input', async () => {
+      const args = [''];
+      const command = CreateGoogleLinkCommand(context);
+
+      await command.handleExecution(args, slackContext);
+
+      expect(slackContext.say.calledWith(sinon.match.string)).to.be.true;
```
Comment (iuliag) [inline test/support/slack/commands/create-google-link.test.js:84]: should it check at least something from the usage string?

**Review by iuliag — APPROVED** [review]
Some minor observations

File: test/support/slack/commands/create-google-link.test.js (hunk 1 above)
Comment (dzehnder) [reply test/support/slack/commands/create-google-link.test.js:41]: oh thats a copy paste error, thanks

**Pull request merged**